generators/express/templates/base/src/infrastructure/database/repositories/UserRepository.ts

```diff
@@ -19,6 +19,10 @@ class DatabaseUserRepository implements UserRepository {
     findByEmail(email: string): Promise<User | null> {
         return this.#dataSource.manager.findOneBy(User, { email });
     }
+
+    save(user: User): Promise<User> {
+        return this.#dataSource.manager.save(user);
+    }
```

Comment:
Do we really need to return the user here ? We could just return `Promise<void>`